src/app/app.html

```diff
@@ -1,14 +1,17 @@
 <ion-split-pane>
   <ion-menu [content]="content">
-    <ion-header aria-hidden="true">
+    <ion-header>
       <ion-toolbar color="primary">
-        <ion-title>PVTrAck</ion-title>
+        <ion-title>
+          PVTrAck
```

Comment:
Yes. I don't see why we have to have it hidden. I can change the pronunciation. But I think I'll hide it again and have the menuClose shown.